frontend/packages/console-app/src/components/nodes/NodesPage.tsx

```diff
@@ -224,10 +249,6 @@ const fetchNodeMetrics = (): Promise<NodeMetrics> => {
       key: 'usedMemory',
       query: 'sum by (instance) (node_memory_MemTotal_bytes - node_memory_MemAvailable_bytes)',
     },
-    {
-      key: 'totalMemory',
-      query: 'sum by (instance) (node_memory_MemTotal_bytes)',
-    },
```

Comment:
https://github.com/openshift/console/blob/master/frontend/public/actions/ui.ts#L91
I think this is the only page that uses this.